src/uwtools/resources/jsonschema/workflow.jsonschema

```diff
@@ -1,101 +0,0 @@
-{
-  "description": "This document is to validate config files from SRW, HAFS, Global",
-  "properties": {
-    "cpl_aqm_parm": {
-      "description": "attributes of coupled air quality",
-      "properties": {
-        "AQM_BIO_DIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "AQM_CONFIG_DIR": {
-          "format": "uri",
-          "type": "string"
-        }
-      },
-      "type": "object"
-    },
-    "platform": {
-      "description": "attributes of the platform",
-      "properties": {
-        "CCPA_OBS_DIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "DOMAIN_PREGEN_BASEDIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "METPLUS_PATH": {
-          "format": "uri",
-          "type": "string"
-        },
-        "MET_BIN_EXEC": {
-          "type": "string"
-        },
-        "MET_INSTALL_DIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "MRMS_OBS_DIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "NCORES_PER_NODE": {
-          "type": "number"
-        },
-        "NDAS_OBS_DIR": {
-          "format": "uri",
-          "type": "string"
-        },
-        "PARTITION_DEFAULT": {
-          "type": "string"
-        },
-        "PARTITION_FCST": {
-          "type": "string"
-        },
-        "PARTITION_HPSS": {
-          "type": "string"
-        },
-        "QUEUE_DEFAULT": {
-          "type": "string"
-        },
-        "QUEUE_FCST": {
-          "type": "string"
-        },
-        "QUEUE_HPSS": {
-          "type": "string"
-        },
-        "SCHED": {
-          "enum": [
-            "slurm",
-            "pbspro",
-            "lsf",
-            "lsfcray",
-            "none"
-          ],
-          "type": "string"
-        },
-        "WORKFLOW_MANAGER": {
-          "enum": [
-            "rocoto",
-            "none"
-          ],
-          "type": "string"
-        }
-      },
-      "type": "object"
-    },
-    "task_get_da_obs": {
-      "description": "task for data assimilation",
-      "properties": {
-        "OBS_SUFFIX": {
-          "type": "string"
-        }
-      },
-      "type": "object"
-    }
-  },
-  "title": "workflow config",
-  "type": "object"
-}
```

Comment:
I think this file was OBE and related to code that was removed in an earlier PR.